apps/solution-e2e/src/support/app.po.ts

```diff
@@ -1 +1,11 @@
 export const getGreeting = () => cy.get('h1');
+export const getSearchBar = () => cy.get('[data-cy="searchbar"]');
+export const findPersonPanel = (id) => cy.get(`[data-testid=${id}]`)
```

Comment:
Actually not your fault, but using the ID only as test-id is probably not a good idea, now I see it...
we probably should use something like "person-panel-${Id} " or something like that, just to discriminate with different components which would use the same person (eg person-row, person-slide and so on)
What do you think about it?

Reply:
Yeah, the cypress doc seem to say that you need to use the data-something that is related to what you're testing

Reply:
I changed it to a person-panel-id as you proposed